### PR TITLE
Set dark theme colors for auth components

### PR DESCRIPTION
**Pull request by AndrewFerr**
Fixes https://github.com/vector-im/element-web/issues/18865

Signed-off-by: Andrew Ferrazzutti <fair@miscworks.net>

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://613b016c06cbd173c97295e9--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Review by gsouquet — APPROVED** [review]
Thank you for opening your first PR for Element 🎉 

It looks good to me from a code perspective. I've added the design team for a review

File: res/themes/dark/css/_dark.scss [hunk 1]
```diff
@@ -184,6 +184,9 @@ $visual-bell-bg-color: #800;
 
 $room-warning-bg-color: $header-panel-bg-color;
 
+$authpage-body-bg-color: $background;
+$authpage-primary-color: $text-primary-color;
```
Comment (SimonBrandner) [inline res/themes/dark/css/_dark.scss:188]: I think `$primary-content` could be better here or one of the colors at the top of the file

**Review by SimonBrandner — COMMENTED** [review]
Thank you for working on this!

**Review by SimonBrandner — APPROVED** [review]
Looks good code-wise, thanks! Could you please add a screenshot to aid design review?

**Comment by AndrewFerr** [comment]
> Could you please add a screenshot to aid design review?

Before code change:
darkAuthOld

After:
darkAuthNew

Before with custom dark colours in `config.json`:
customDarkAuthOld

After:
customDarkAuthNew

**Review by janogarcia — APPROVED** [review]
Thanks for the contribution, @AndrewFerr ! 👍

**Pull request merged**